src/test/java/com/aws/greengrass/lifecyclemanager/LogManagerHelperTest.java

```diff
@@ -123,9 +131,122 @@ void GIVEN_mock_service_WHEN_getComponentLogger_THEN_logs_to_correct_log_file()
         assertEquals(0, ggLogFile.length());
     }
 
+    @Test
+    void GIVEN_mock_service_logger_WHEN_reconfigure_THEN_change_applied_correctly() throws IOException {
+        Path tempRootDir2 = tempRootDir.resolve("2");
+        Path tempRootDir3 = tempRootDir.resolve("3");
+        String mockServiceName = "MockService001";
+        when(mockGreengrassService.getServiceName()).thenReturn(mockServiceName);
+        LogConfig.getInstance().setStore(LogStore.FILE);
+        Logger componentLogger = LogManagerHelper.getComponentLogger(mockGreengrassService);
+
+        // change log file size
+        LogConfigUpdate newConfig = LogConfigUpdate.builder().fileSizeKB(1L).build();
+        LogManager.reconfigureAllLoggers(newConfig);
+
+        // should apply change to all loggers
+        LogConfig testLogConfig = LogManager.getLogConfigurations().get(mockServiceName);
+        assertEquals(1, testLogConfig.getFileSizeKB());
+        assertEquals(1, LogManager.getRootLogConfiguration().getFileSizeKB());
+        assertEquals(1, LogManager.getTelemetryConfig().getFileSizeKB());
+
+        // log less than 1k bytes, should not rotate
+        // 1 message of size 50 is about 128 bytes in TEXT format
```

Comment:
nit - move this calculation of "size 50 ->  128 bytes" into the helper method if possible.

The interface should just be able to `log(int bytes)` so that you could easily control how many bytes from here without additional layer of calculation

Reply:
made it exact!